nursery/reference-doh.yml

```diff
@@ -0,0 +1,75 @@
+rule:
+  meta:
+    name: reference of DoH
+    namespace: communication/dns
+    author: markus.neis@swisscom.com
+    scope: file
+    references:
+      - https://github.com/curl/curl/wiki/DNS-over-HTTPS
+    examples:
+      # - 749e7becf00fccc6dff324a83976dc0d
+  features:
+    - or:
+      - string: /.*https://doh.seby.io:8443/dns-query.*/i
```

Comment:
based on your example, i see that we do need to use regex here to account for application data being passed in the url. 

do you think its reasonable to remove the wildcard at the start of each of the strings here? regex matching is one area that capa is not very performant, and i think having a static anchor at the start of each pattern can maybe maintain performance as we add lots of patterns. 

![image](https://user-images.githubusercontent.com/156560/96896464-c3338100-144a-11eb-8365-b22fa22140ba.png)

Reply:
We decided to use `re.search` to make rule writing more convenient.
https://github.com/fireeye/capa/blob/d5186f160dd461eb4fa779d9dc75f0766aedbf4f/capa/features/__init__.py#L151-L155

Reply:
removed the wildcard at the start but apparently not necessary thanks @mr-tz